index.rst

```diff
@@ -1,7 +1,7 @@
 .. Gamify IT documentation master file, created by
-   sphinx-quickstart on Thu May 26 13:13:28 2022.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+sphinx-quickstart on Thu May 26 13:13:28 2022.
+You can adapt this file completely to your liking, but it should at least
+contain the root `toctree` directive.
```

Comment:
Probably not a good idea.
I think you just uncommented this block with this action, but I'm also no expert on RST.